src/gps_time_listener.rs

```diff
@@ -0,0 +1,65 @@
+use chrono::offset::Utc;
+use regex::Regex;
+use yobot::listener::{Message, MessageListener};
+
+/** Unix timestamp of the GPS epoch 1980-01-06 00:00:00 UTC */
+const GPS_EPOCH: i64 = 315964800;
+
+/** Number of seconds in a minute. */
+const MINUTE_SECS: i64 = 60;
+
+/** Number of minutes in an hour. */
+const HOUR_MINUTES: i64 = 60;
+
+/** Number of hours in a day. */
+const DAY_HOURS: i64 = 24;
+
+/** Number of seconds in a day. */
+const DAY_SECS: i64 = (DAY_HOURS * HOUR_MINUTES * MINUTE_SECS);
+
+const WEEK_DAYS: i64 = 7;
+
+/** Number of seconds in a week. */
+const WEEK_SECS: i64 = (WEEK_DAYS * DAY_SECS);
+
+pub struct GpsTimeListener {
+    regex: Regex,
+}
+
+impl GpsTimeListener {
+    pub fn new() -> GpsTimeListener {
+        GpsTimeListener {
+            regex: Regex::new(r"^Time$").unwrap(),
+        }
+    }
+}
+
+impl MessageListener for GpsTimeListener {
+    fn help(&self) -> String {
+        String::from("Converts current time into GPS time. No muss, no fuss.")
```

Comment:
other help strings use backticks, it's pretty nitty but could update as:
```suggestion
        String::from("`time`: Converts current time into GPS time. No muss, no fuss.")
```

Reply:
Done